front/src/applications/operationalStudies/components/SimulationResults/SimulationResultsMap/__tests__/TrainHoverPosition.spec.ts

```diff
@@ -79,8 +79,8 @@ describe('makeDisplayedHeadAndTail', () => {
         pathLineString,
         sideDimensions
       );
-      expect(head).toEqual(0.9);
-      expect(tail).toEqual(0.1);
+      expect(head).toBe(0.9);
```

Comment:
Keep toEqual with floating point.
If it’s doing this kind of stuff, `vitest/prefer-to-be` must be removed.


![Screenshot_20230224_112039-1](https://user-images.githubusercontent.com/24652545/221154659-22ac06a2-1479-4ee5-bfcf-fa84a148cdd6.png)

Reply:
Done